src/app/components/drawer/SavedSearches/SavedSearchActions.js

```diff
@@ -243,14 +297,14 @@ const ProjectActions = ({
           <FormattedMessage
             defaultMessage="Delete list"
             description="'Delete' here is an infinitive verb"
-            id="projectsComponent.deleteType"
+            id="savedSearchActions.deleteType"
```

Comment:
Please let's keep existring string ids as they were to avoid unnecessary re-translation work and costs.

Reply:
Done